Update/omot_016.txt

```diff
@@ -957,7 +957,7 @@ void main()
 			NULL, "\"Right.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600627", 256, TRUE);
 	OutputLine(NULL, "鷹野さんの理論は突拍子もない話ばかりですが、案外この連続事件の裏にあるもの、もし偶然じゃなかったら一筋縄ではいきそうにないですよ」",
-			NULL, " Takano-san's theories are just crazy stories, but surprisingly enough, if this series of events isn't a coincidence, they do seem to point toward that.\"", Line_Normal);
+			NULL, " Takano-san's theories are just crazy stories, but if this series of events isn't just a coincidence, we'll have a harder time taking the wraps off than we're thinking now.\"", Line_Normal);
```

Comment:
Allow me to rephrase this.
"Takano-san's theories are just crazy stories, but if this series of events isn't just a coincidence, we'll have an unexpectedly hard time taking the wraps off."